frontend/src/features/admin-form/settings/SettingsEndPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Skeleton } from '@chakra-ui/react'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EndPageSettingsInput } from './components/EndPageSettingsSection/EndPageSettingsInput'
+import { useAdminFormSettings } from './queries'
+
+export const SettingsEndPage = (): JSX.Element => {
+  const { data: settings, isLoading } = useAdminFormSettings()
```

Comment:
as mentioned we shouldn't think of endPage as part of settings, so maybe just use `useAdminForm`?

Reply:
Sounds good. Updated the PR to use `useAdminForm`